catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/cache/solr/impl/CachingFederationStrategy.java

```diff
@@ -269,15 +248,17 @@ private QueryResponse sourceFederate(List<Source> sources, final QueryRequest qu
                         logger.warn("Plugin stopped processing", e);
                     }
 
-                    finalQueryRequest = modifiedQueryRequest;
                     if (source instanceof CatalogProvider && SystemInfo.getSiteName()
                             .equals(source.getId())) {
-                        finalQueryRequest = getQueryRequestWithAdditionalFilters(
-                                modifiedQueryRequest);
+                        sourceQueryRequest =
```

Comment:
Could do that or make a second prefed plugin but...I had started out making a separate implementation of the prefed plugin for validation but the `showErrors` and `showWarnings` properties wouldn't be readily available to it. I felt that the extra effort of moving their configuration around was outside the scope of this ticket.